package.json

```diff
@@ -34,30 +34,30 @@
     "start": "gulp"
   },
   "devDependencies": {
-    "autoprefixer": "^10.4.2",
+    "autoprefixer": "^10.4.13",
     "browserslist-config-nebenan": "^1.0.0",
-    "chalk": "^4.1.0",
-    "eslint": "^7.6.0",
-    "eslint-config-nebenan": "^3.1.1",
+    "chalk": "^4.1.2",
+    "eslint": "^8.29.0",
+    "eslint-config-nebenan": "^4.4.0",
     "fontellizr": "^1.4.4",
     "gulp": "^4.0.2",
     "gulp-livereload": "^4.0.2",
     "gulp-postcss": "^9.0.1",
-    "gulp-pug": "^4.0.1",
+    "gulp-pug": "^5.0.0",
     "gulp-sass": "^5.1.0",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-webserver": "^0.9.1",
     "node-sass-glob-importer": "^5.3.2",
-    "postcss": "^8.4.5",
+    "postcss": "^8.4.20",
     "postcss-flexbugs-fixes": "^5.0.2",
     "promised-del": "^1.0.2",
-    "sass": "^1.49.0",
+    "sass": "^1.56.2",
```

Comment:
maybe check if it still successfully builds with sass `1.34.0` just to be sure? we list that version in peer deps and don't want to update it (because it would cause a breaking change in ui-kit)

Reply:
fixed the sass version to 1.34.0 and the preview still build successfully. 

what was the breaking change?

Reply:
> fixed the sass version to 1.34.0 and the preview still build successfully.

Ok awesome, then we're good.

> what was the breaking change?

No idea. I don't trust sass semantic versioning. They do breaking changes in minor version bumps, really weird